queue/queue.py

```diff
@@ -8,10 +8,13 @@ def __init__(self):
     self.storage = LinkedList()
 
   def enqueue(self, item):
-    pass
+    self.size +=1
+    self.storage.add_to_tail(item)
   
   def dequeue(self):
-    pass
+    if self.size > 0:
+      self.size -=1
+      return self.storage.remove_head()
```

Comment:
This is a really efficient solution, I wouldn't have thought to do it this way. What other ways can you implement this?